script/go

```diff
@@ -2,10 +2,6 @@
 
 set -e
 
-. script/bootstrap
+. script/common
 
-mkdir -p bin
-bindir="$PWD"/bin
-
-cd .gopath/src/github.com/github/gh-ost
-go "$@"
+$DGO go "$@"
```

Comment:
This still felt somewhat useful. Shouldn't be necessary if you have go installed locally, but :man_shrugging: